### DISCUSSION TITLE
Publish packages to Docker Hub

### DISCUSSION THREAD

**Discussion by wdhdev** [diced/zipline#606] [Ideas]
I think it would be helpful if you also published Docker images to https://hub.docker.com as well as ghcr.io as this would give users an alternative option to pull from.

The main motivation behind this idea is I personally run Zipline on an IPv6 only machine (I do not require an IPv4 for it as I run it behind CF proxy) which is unable to pull from ghcr.io without using a GitHub IPv6 proxy such as https://danwin1210.de/github-ipv6-proxy.php due to GitHub / ghcr.io not supporting IPv6.

I hope this idea will be considered and I am happy to help implement it if needed :)

**Comment by oie1** [comment]
solution: dont use docker

**Reply by wdhdev** [reply]
Installing from source requires cloning of the repository from GitHub, which does infact not support IPv6.

**Reply by oie1** [reply]
Or maybe, just maybe, don't use dumbass ipv6

**Reply by oie1** [reply]
ipv6 is shittier in every way

**Comment by wdhdev** [comment]
Added in #613.
